app/src/main/res/navigation/navigation_graph.xml

```diff
@@ -59,4 +59,12 @@
         android:id="@+id/orderCompletedFragment"
         android:name="org.fossasia.openevent.general.order.OrderCompletedFragment"
         android:label="OrderCompletedFragment"/>
+    <fragment
+        android:id="@+id/navigation_login"
```

Comment:
Should be cameCase

Reply:
Yes, but it should be same with menu item ids for bottom navigation. Need to change all menu item ids for this. Should I change all menu item ids for this?

Reply:
All ids that are referenced in code should be in camelCase

Reply:
Updated